features/telemetry/protocols/sflow/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/sflow/parser/proto/headers/Inet4Header.java

```diff
@@ -121,20 +122,28 @@ public Inet4Header(final int tos, final int totalLength, final int protocol, fin
         this.tcpFlags = tcpFlags;
     }
 
-    public void writeBson(final BsonWriter bsonWriter) {
+    public Inet4Address getSrcAddress() {
+        return srcAddress;
+    }
+
+    public Inet4Address getDstAddress() {
+        return dstAddress;
+    }
+
+    public void writeBson(final BsonWriter bsonWriter, final SampleDatagramEnrichment svcs) {
```

Comment:
What does `svcs` stand for?

Reply:
The class was called something else before I finally renamed it to SampleDatagramEnrichment. Fixed.